moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useAdvancedSearch.js

```diff
@@ -62,12 +57,17 @@ const makeAdvancedSearchWhereString = (filters) =>
       }
       return `${field}: { ${gqlOperator}: ${value} }`;
     })
-    .filter((value) => value !== null)
-    .join(", ");
+    .filter((value) => value !== null);
 
 export const useAdvancedSearch = () => {
-  const filterQuery = useFilterQuery(useLocation().search);
-  const initialFilterState = useMakeFilterState(filterQuery);
+  /* Get advanced filters settings from search params if they exist */
+  let [searchParams] = useSearchParams();
+  const initialFilterState = useMakeFilterState(searchParams);
+
+  /* Determine or/any from search params if it exists */
+  const isOrFromSearchParams = searchParams.get("isOr");
+  const initialIsOrState = isOrFromSearchParams === "true" ? true : false;
```

Comment:
✅ i tried to break this feature by setting the url param to `isOr=pizza` and i observed that this was interpreted as the  "match all" setting